{"name": "Edit Spend With Error", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestsEditeSpends' object has no attribute 'reload'", "trace": "page = <tests.test_main.TestsEditeSpends object at 0x000001B8F2829810>\ncreated_spend = SpendCreate(id='47c6b620-be46-4469-a6c9-384315e7be14', spendDate=datetime.datetime(2025, 7, 6, 11, 11, 48, 242000, tzi...ame='asd', category=Category(id='b9edef84-f11a-4a7c-b984-96431aa3c426', name='street', username='asd', archived=False))\n\n    @allure.story(\"Редактирование траты с ошибкой\")\n    def test_edit_spend_with_error(page: Page, created_spend):\n    \n        now = datetime.now(timezone.utc)\n        spending_date = now.strftime(\"%Y-%m-%dT%H:%M:%S.%f\")[:-3] + \"Z\"\n    \n>       page.reload()\n        ^^^^^^^^^^^\nE       AttributeError: 'TestsEditeSpends' object has no attribute 'reload'\n\ntests\\test_main.py:113: AttributeError"}, "attachments": [{"name": "stdout", "source": "e22bbb2f-5929-427c-8f7c-a59f75e385ad-attachment.txt", "type": "text/plain"}], "start": 1751800308271, "stop": 1751800308272, "uuid": "54fd511b-4053-47ce-be59-eef1a4fad708", "historyId": "00a1d3fff5dfa34e67a0269b1c5f64db", "testCaseId": "00a1d3fff5dfa34e67a0269b1c5f64db", "fullName": "tests.test_main.TestsEditeSpends#test_edit_spend_with_error", "labels": [{"name": "story", "value": "Редактирование траты с ошибкой"}, {"name": "epic", "value": "Страница трат"}, {"name": "feature", "value": "Редактированиие трат"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "TestsEditeSpends"}, {"name": "host", "value": "WIN-V8TQK9SAURU"}, {"name": "thread", "value": "25556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_main"}]}